contributors/design-proposals/bootstrap-discovery.md

```diff
@@ -70,16 +70,16 @@ Note that TLS bootstrap (which establishes a way for a client to authenticate it
 If the ClusterInfo information is hosted in a trusted place via HTTPS you can just request it that way.  This will use the root certificates that are installed on the system.  It may or may not be appropriate based on the user's constraints.
```

Comment:
It is defined in the section called "Cluster location information" but isn't as clear as it could be.  I'll add more text there and call out the term.